### ISSUE TITLE
update() for Table and InlineTable yield surprising result

### ISSUE THREAD

**Issue by uranusjr** [sdispater/tomlkit#8]
```pycon
>>> import tomlkit
>>> t = tomlkit.table()
>>> t.update({'a': 1})
>>> t
{'a': 1}
>>> t.as_string()
''
```

I think the solution is to implement them to update `_value`? I’d be interested to work on this (and probably some other inherited methods when I find them) if they are considered acceptable additions.

**Comment by sdispater** [comment]
Yes, tables/inline tables should behave exactly like a dict and, if not, it's a bug :-)

You can take a shot at it if you want.

**Comment by sdispater** [comment]
This is fixed in the latest `0.4.3` release.
